integrationTests/mock/unsignedTxHandlerMock.go

```diff
@@ -47,7 +55,7 @@ func (ut *UnsignedTxHandlerMock) CleanProcessedUTxs() {
 }
 
 // ProcessTransactionFee -
-func (ut *UnsignedTxHandlerMock) ProcessTransactionFee(cost *big.Int) {
+func (ut *UnsignedTxHandlerMock) ProcessTransactionFee(cost *big.Int, txHash []byte) {
 	if ut.ProcessTransactionFeeCalled == nil {
```

Comment:
Apply the same signature to the mock method ProcessTransactionFeeCalled

Reply:
done